tests/link/test_jax.py

```diff
@@ -965,3 +968,25 @@ def test_extra_ops():
     )
     fgraph = FunctionGraph([], [out])
     compare_jax_and_py(fgraph, [])
+
+
+@pytest.mark.xfail(reason="The RNG states are not 1:1", raises=AssertionError)
+def test_random():
+    rng = shared(np.random.RandomState(123))
+    out = normal(rng=rng)
+    fgraph = FunctionGraph([out.owner.inputs[0]], [out], clone=False)
+    compare_jax_and_py(fgraph, [])
+
+
+@pytest.mark.xfail(reason="The RNG states are not 1:1", raises=AssertionError)
+def test_randomstream():
+    srng = RandomStream(seed=123)
+    out = srng.normal()
+    fgraph = FunctionGraph([out.owner.inputs[0]], [out])
+    compare_jax_and_py(fgraph, [])
+
+    new_seed = 43298
+    srng.seed(new_seed)
+    out = srng.normal()
+    fgraph = FunctionGraph([out.owner.inputs[0]], [out])
+    compare_jax_and_py(fgraph, [])
```

Comment:
We should be able to devise some non-`xfail` tests.  For instance, we can verify that the RNG states work as expected (i.e. that we can produce the same results for the same states).

Reply:
I cannot figure out how to give input directly to and take output from the `RandomVariable` instances like `normal` while them being a `FunctionGraph`. So I may require some help with that.